Analysis/src/EventTags.cpp

```diff
@@ -78,15 +84,22 @@ EventTags EventTagCreator::CreateEventTags(const DataId& dataId_base, float weig
         { DiscriminatorWP::Medium, weight_btag_Medium },
         { DiscriminatorWP::Tight, weight_btag_Tight },
     };
-    const auto get_weight_btag = [&](DiscriminatorWP wp, UncertaintySource unc_source, UncertaintyScale unc_scale) {
+
+    const auto get_weight_btag = [&](DiscriminatorWP wp, UncertaintySource unc_source, UncertaintyScale unc_scale, bool tune) {
```

Comment:
`tune` is already caught by `[&]`, no need to pass it as an argument to the lambda.